Cargo.toml

```diff
@@ -1,7 +1,7 @@
 # extend if new rust crates are added so rust analyzer isnt confused
 [workspace]
 members = [
-    "x/programs/rust/expose_macro",
+    "x/programs/rust/public_macro",
```

Comment:
This should be `sdk_macros` since you can create multiple macros in here. Best leave it open-ended so it doesn't need to be renamed again in the future.